examples/rust/BUILD

```diff
@@ -8,8 +8,8 @@ load("@io_bazel_rules_docker//rust:image.bzl", "rust_image")
 
 # NOTE: Bazel Rust rules don't support cross-compilation yet,
 # so at this time it's required that you build on the same platform as the image
-rust_image(
-    name = "rust_example",
-    srcs = ["src/main.rs"],
-    base = "//cc:cc_debian10",
-)
+# rust_image(
```

Comment:
I forgot about this.  I'll have to see if tags solves the problem here as well :(